src/Modal/Modal.js

```diff
@@ -185,27 +185,20 @@ Modal.Header = ({ title, children, showClose = true }) => {
     <ModalHeader>
       <ModalHeaderInner>
         <Flex alignItems="center">
-          {title && (
-            <Modal.Title role="heading" tabIndex="0">
-              {title}
-            </Modal.Title>
-          )}
+          {title && <Modal.Title role="heading">{title}</Modal.Title>}
```

Comment:
intentional to remove the tab-index?

Reply:
Yes. So it doesn't take the focus when the content of the modal has something that needs autofocus.